tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Remote Staking via XCM
```

Comment:
Maybe in these tutorials we can add something like the date an the name of the guy who wrote it in italics (cc @eshaben not sure what you think. Similar to what is used in the Blogpost section of the website

`December XX, YYYY | by Kevin`